pkg/terminal/terminal.go

```diff
@@ -0,0 +1,22 @@
+package terminal
+
+import (
+	"os"
+
+	"golang.org/x/crypto/ssh/terminal"
```

Comment:
This package is deprecated, use `golang.org/x/term`

Reply:
good idea